### DISCUSSION TITLE
Spacing should take string values

### DISCUSSION THREAD

**Discussion by activenode** [mantinedev/mantine#3168] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

When re-defining the spacing in the theme it should allow `string` types as well. Indeed it does work with string-types but TypeScript complains because the type isn't set (see Sandbox).

`spacing: {xs: '2rem'}` should be allowed but isn't.

### What version of @mantine/hooks page do you have in package.json?

5.9.1

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/tender-forest-lue6tn?from-embed=&file=/src/App.tsx

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Changing the type accordingly.

**Comment by Cuzart** [comment]
This also applies to radius and fontSizes in the theme object as they also are typed as MantineSIzes.
